src/docs/reference/modules/Media.Azure/README.md

```diff
@@ -1,6 +1,8 @@
-# Azure Media Storage (`OrchardCore.Media.Azure`)
+# Microsoft Azure Media (`OrchardCore.Media.Azure`)
 
-The Azure Media Storage feature enables support for storing assets in Microsoft Azure Blob Storage.
+The Microsoft Azure Media module enables support for storing assets in Microsoft Azure Blob Storage.
+
+## Azure Media Storage (`OrchardCore.Media.Azure.Storage`)
```

Comment:
Either add `Mirosoft` here or remove `Microsoft` from line 1.

Reply:
The module's manifest contains "Microsoft Azure Media" for the module name too, though not for the Storage (and now ImageSharp) feature. 

Microsoft uses "Microsoft Azure" and just "Azure" interchangeably, and we use just "Azure" in the UI and code. So, let's go with that.